stores/sql/main.go

```diff
@@ -305,6 +305,77 @@ func CopyObject(ctx context.Context, tx sql.Tx, srcBucket, dstBucket, srcKey, ds
 	return fetchMetadata(dstObjID)
 }
 
+func DeleteHostSector(ctx context.Context, tx sql.Tx, hk types.PublicKey, root types.Hash256) (int, error) {
+	// update the latest_host field of the sector
+	_, err := tx.Exec(ctx, `
+		UPDATE sectors
+		SET latest_host = (
+			SELECT * FROM (
+				SELECT h.public_key
```

Comment:
We could sort these, we don't have a score field but there's a couple of fields that might be interesting. I'm on the fence on whether this is prematurely optimising things, it's virtually free, we know redundancy is quite high, so I think it probably doesn't hurt trying to select a good host here.

Reply:
Score is not useful here. The only thing that would help is a timestamp on when a sector was linked to a contract and select the most recent one that we migrated to. Unfortunately we don't have that  and it seemed a bit overkill to add a migration just for that in this PR. I'm also not sure if it really makes a difference in a real world use case or if it just uses a ton of space for another column on a table with potentially a hundred million rows.